client/packages/android/capacitor.config.ts

```diff
@@ -0,0 +1,31 @@
+import { CapacitorConfig } from '@capacitor/cli';
```

Comment:
I moved this file from client (also moved plugin and corodva-android dependencies to packages/android, to isolate it a bit more from client dir and keep related package configs within the package)